docs/general/installation/linux.md

```diff
@@ -96,7 +96,13 @@ However [`rpmfusion`](https://rpmfusion.org/) provides both `jellyfin-server` an
 
    :::
 
-6. Go to `localhost:8096` or `ip-address-of-jellyfin-server:8096` to finish setup in the web UI
+6. Reload the firewall to apply the new rules:
+
+   ```sh
+   sudo firewall-cmd --list-services
```

Comment:
I don't think this is the right command for reloading firewalld

Reply:
Correct one should be `sudo firewall-cmd --reload`

Reply:
wow- i do know that :joy:  

and i dont know how I pushed this, must have been in a rush